CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ for further information about branching and tagging conventions.
 ## [Unreleased]
 #### Fixed
 - A bug with localization
+- A bug with incorrect display of the amount of the maximum issued asset in
```

Comment:
A bug with incorrect displaying `issuedAmount` in `AssetRequests.Show` component

Reply:
done